ArduCopter/mode_guided.cpp

```diff
@@ -468,6 +471,15 @@ bool ModeGuided::stabilizing_vel_xy() const
     return !((copter.g2.guided_options.get() & uint32_t(Options::DoNotStabilizeVelocityXY)) != 0);
 }
 
+// set guided mode body-frame rates
+void ModeGuided::set_rate_bf_roll_pitch_yaw(float roll_rate_bf_cds, float pitch_rate_bf_cds, float yaw_rate_bf_cds, bool use_body_rate)
+{
+    guided_angle_state.use_body_rate = use_body_rate;
+    guided_angle_state.roll_rate_cds = ToDeg(roll_rate_bf_cds) * 100.0f;
```

Comment:
This conversion makes it seem like the "roll_rate_bf_cds" argument should be renamed to "roll_rate"bf_rads" (and same for pitch)

Reply:
The arguments are renamed in ac09fdc.